test/parallel/test-child-process-net-reuseport.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const common = require('../common');
+const { checkSupportReusePort, options } = require('../common/net');
+const assert = require('assert');
+const child_process = require('child_process');
+const net = require('net');
+
+if (!process.env.isWorker) {
+  checkSupportReusePort().then(() => {
+    const server = net.createServer();
+    server.listen(options, common.mustCall(() => {
+      const port = server.address().port;
+      const workerOptions = { env: { ...process.env, isWorker: 1, port } };
+      let count = 2;
+      for (let i = 0; i < 2; i++) {
+        const worker = child_process.fork(__filename, workerOptions);
+        worker.on('exit', common.mustCall((code) => {
+          assert.strictEqual(code, 0);
+          if (--count === 0) {
+            server.close();
+          }
+        }));
+      }
+    }));
+    server.on('error', common.mustNotCall());
```

Comment:
Any reason for keeping this? An unhandled `'error'` error event will make the process exit anyway.

Reply:
Yes, you are right, have removed the code.